src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the image `../images/star.svg` exists at the specified path. If not, update the path or add the image to the directory.